tests/tablespaces/test_tablespaces_05.py

```diff
@@ -0,0 +1,15 @@
+from nose.tools import *
+from tests.tablespaces.tablespace_utils import node1
+from tests.tablespaces.tablespace_utils import node2
+
+
```

Comment:
Same as the pause comment, a comment about the Makefile running unpause is necessary here.